src/priya/Program2Exam4.java

```diff
@@ -0,0 +1,42 @@
+//Program 2: WAP to find two String are identical or not
+//input 1:String s1 = "technoCredits";
+//String s2 = "teChnoCreditS";///output 1: both string are not identical
+//input 2:String s1 = "pune";
+//String s2 = "pune";////output 2: both string are identical
+
+package priya;
+
+public class Program2Exam4 {
+
+	void identicalString(String s1, String s2) {
+		boolean flag = false;
+		boolean word = false;
+		for (int i = 0; i < s1.length(); i++) {
+			word = false;
+			if (s1.length() == s2.length()) {
+				flag = true;
+				if (s1.charAt(i) == s2.charAt(i)) {
+					word = true;
```

Comment:
you should write != condition which reduce number of line, once condition satisfied then put break,.